templates/pre_unsubscribe.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.4/css/bulma.min.css">
+    <link rel="stylesheet" href="{{url_for('static', filename='styles.css')}}">
+    <link href="https://cdn.jsdelivr.net/npm/hint.css@3.0.0/hint.min.css" rel="stylesheet">
+    <script>
+        function handleMenuChange() {
+            var mainMenu = document.getElementById('main-menu');
+            var unsubscribeOptions = document.getElementById('unsubscribe-options');
+            var submitButton = document.getElementById('submit-button');
+            
+            // If 'Unsubscribe from emails' is selected, show submenu and disable submit
+            if (mainMenu.value === 'unsubscribe-emails') {
+                unsubscribeOptions.style.display = 'block';
+                submitButton.disabled = true; // Disable submit button initially
+            } else {
+                unsubscribeOptions.style.display = 'none';
+                submitButton.disabled = false; // Enable submit button for other options
+            }
+        }
+    
+        function handleSubmenuChange() {
+            var unsubscribeMenu = document.getElementById('unsubscribe-menu');
+            var submitButton = document.getElementById('submit-button');
+            
+            // Enable submit button if a valid option is selected in the submenu
+            if (unsubscribeMenu.value !== '') {
+                submitButton.disabled = false;
+            } else {
+                submitButton.disabled = true;
+            }
+        }
+    </script>
+    <title>POPROX Newsletter</title>
+	{% block header %}
```

Comment:
?

Reply:
I'm thinking this might be a copy/paste from layout (or another page which copy/pasted from layout and didn't get caught)
This code is not reasonable here. 

Ultimately, the fix here is to make this file inherit from template.html as well.